operations/loyaltyprograms.md

```diff
@@ -85,4 +85,153 @@ Note this operation uses [Pagination](../guidelines/pagination.md).
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the loyalty program. |
 | `Name` | string | required | Name of the loyalty program. |
-| `Code` | string | required | Code of the loyalty program. |
\ No newline at end of file
+| `Code` | string | required | Code of the loyalty program. |
+
+## Add loyalty programs
+
+Adds loyalty programs to the enterprise (in the given activity state).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/loyaltyPrograms/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "LoyaltyPrograms": [
+        {
+            "DataClusterId": "8ddea57b-6a5c-4eec-8c4c-24467dce118e",
+            "Name": "Platinum Club",
+            "Code": "PC01"
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `LoyaltyPrograms` | array of [Loyalty program parameters](#loyalty-program-parameters) | required, max 1000 items | Loyalty programs to be added. |
+
+### Loyalty program parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `DataClusterId` | string | required | Unique identifier of the chain whose member the enterprise is. |
+| `Name` | string | required | Name of the loyalty program. |
+| `Code` | string | required | Code of the loyalty program. |
+
+### Response
+
+```javascript
+{
+    "LoyaltyPrograms": [
+		{
+            "Id": "a58ff7cb-77e3-495a-bd61-aecf00a3f19d",
+            "Name": "Platinum Club",
+            "Code": "PC01"
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `LoyaltyPrograms` | array of [Loyalty program](#loyalty-program) | required | Added loyalty programs. |
+
+## Update loyalty programs
+
+Updates information about the specified [Loyalty program](#loyalty-program).
```

Comment:
Consistency: In other operations you don't have a link to the entity. Personally I don't think it's necessary since we're on the same page.